app/helpers/hosts_helper.rb

```diff
@@ -492,6 +492,8 @@ def build_error_link(type, id)
   end
 
   def inherited_by_default?(field, host)
+    force = host.force_inherited(field)
```

Comment:
after the refactoring, we need check the `@host` is present, otherwise it doesn't work in hostgroup